docs/storage/multipathing.md

```diff
@@ -134,6 +138,10 @@ linkStyle 13 stroke:#8C8C8C,stroke-width:2px;
 1. On one of the XCP-ng host, make sure that the `multipath.conf` configuration includes your storage equipment.
 
    This can be found in the file `/etc/multipath.xenserver/multipath.conf`
+
+   :::warning
+   Never change the file  `/etc/multipath.xenserver/multipath.conf`.
```

Comment:
* I think it's better english to put word "file" at the end. 
* I'd use verb "modify"
* I like to explain why something is forbidden

Suggestion: ""Do not modify `/etc/multipath.xenserver/multipath.conf` directly, as any changes may be overwritten by future system updates." 

Also applies to the other section.